SECURITY.md

```diff
@@ -27,9 +27,9 @@ The following table provides the status of the Otter versions under support and
 
 | Otter version | Angular version | Support ends |
 |---------------|-----------------|--------------|
-| ^8.2.0        | ^15.0.0         | 2024-05-18   |
-| ^7.6.0        | ^14.0.0         | 2023-11-18   |
-| ^6.3.0        | ^13.0.0         | 2023-05-04   |
+| 9             | ^16.0.0         | 2024-11-08   |
```

Comment:
That's a bit weird to use 2 similar range notation for Otter versions and Angular version.
For me it should be `9 - 16` or `^9.0.0 - ^16.0.0`